pkg/devspace/kubectl/util.go

```diff
@@ -65,15 +66,22 @@ func EnsureDefaultNamespace(config *latest.Config, client kubernetes.Interface,
 	if defaultNamespace != "default" {
```

Comment:
Use the correct kubernetes constant for this

Reply:
Done